modules/rpc/thrift.go

```diff
@@ -36,27 +35,21 @@ var _setupMu sync.Mutex
 type CreateThriftServiceFunc func(svc service.Host) ([]transport.Procedure, error)
 
 // ThriftModule creates a Thrift Module from a service func
-func ThriftModule(hookup CreateThriftServiceFunc, options ...modules.Option) service.ModuleCreateFunc {
```

Comment:
Each module have a specific set of options and they can overlap, passing everything to a builder is very confusing..